app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -26,51 +28,21 @@ export const SIXTY = 60;
 export const EIGHTY = 80;
 export const TWO_HUNDRED = 200;
 
-export const CONTACT_DETAILS_DATA = [
-  {
-    id: 0,
-    name: "Ronald Richards",
-  },
-  {
-    id: 1,
-    name: "Jacob Jones",
-  },
-  {
-    id: 2,
-    name: "Ronald Jones",
-  },
-  {
-    id: 3,
-    name: "Jacob Richards",
-  },
-];
+export const CONTACT_DETAILS_DATA = buildSelectOption([
```

Comment:
You are not building a single option here. You are building options, plural. Thus rename the util method appropriately.

Reply:
Okay. Thank you. Will make the appropriate changes.